Sources/LiveKit/Participant/LocalParticipant.swift

```diff
@@ -421,7 +421,7 @@ extension LocalParticipant {
             }
     }
 
-    func republishTracks() async throws {
+    func rePublishAllTracks() async throws {
```

Comment:
nit: republish does not need to be capitalized like rePublish
```suggestion
    func republishAllTracks() async throws {
```